src/dp.rs

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Differential privacy (DP) primitives.
+use crate::field::FieldElement;
+use crate::vdaf::prg::SeedStream;
+use std::fmt::Debug;
+
+/// Positive rational number to represent DP parameters and noise distributions.
+#[allow(dead_code)]
+#[derive(Clone, Debug)]
+pub struct Rational {
+    numerator: u32,
+    denominator: u32,
+}
+
+impl Rational {
+    #[allow(dead_code)]
```

Comment:
Instead putting a dead_code here, I think we could just way to implement this constructor until we need it.

Reply:
Sounds good. Can I still keep the dead_code for ZeroConcentratedDifferentialPrivacyBudget and DiscreteGaussian? The constructor seems like a good place to keep some documentation about the struct fields.

Reply:
Yup, I meant this one specifically.